tests/html5skin-test.js

```diff
@@ -679,5 +679,17 @@ describe('Controller', function() {
       expect(controller.state.videoQualityOptions.availableBitrates).toBe(qualities.bitrates);
     });
 
+    describe('Toggle fullscreen', function () {
```

Comment:
This `describe` is for tests related to video qualities. This test should be moved to a different section

Reply:
fixed